i18n/en.jsonp.js

```diff
@@ -11,6 +11,7 @@ document.localeJson = {
       "home-btn-random": "Random",
       "home-btn-back": "Back",
       "home-btn-fileselect": "Select ZIM file(s)",
+      "home-btn-library": "Browse ZIM Library",
```

Comment:
Mea culpa, and it's my mistake, but I've just realized that neither the fileselect nor the library buttons are on the Home page. They are on the Confiuration page, so both of these should really be `configure-btn-library` and `configure-btn-fileselect`, and they should be moved directly beneath `configure-about-usage-link` below. Of course, you'd need to do the same for the other language files, and rename the `data-i18n` attribute of both buttons in `index.html`.